docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1370,15 +1370,12 @@ duplicates at this time.
 
 
 **Data Errors/Noise** 
-In the future, we will add a noise function designed to replicate 
-missed or incorrect data. This includes incorrect name spelling, 
-addresses, age or DOB, and person swaps. 
 
-Some errors may be introduced within the simulation, such as 
-duplicate people or swapped people. Others may be introduced 
-at the time of sampling, such as name and address misspellings. 
+.. todo::
+    Engineers: these things are not yet defined and might be edited later - they should be parameterizable 
 
-These are not currently modeled. 
+    - We currently have net undercounts, might want to have omissions and duplicates instead. Having a perfect census for now should be fine. 
```

Comment:
I don't think this clearly defines what needs to be parameterizable. What we need to do is scope: if we want to have the flexibility to add omissions and duplicates, what values will the omission/duplication function need as inputs?

As a rough example (haven't thought this through), if we know we will want some flexibility about duplicating college students, we might need GQ type and tracked parent/guardian.

It's possible that we can make the ask here to be able to use all tracked columns (I don't know how expensive this could get) -- might be an engineering question.

Reply:
Re: the last sentence "Having a perfect census for now should be fine" -- it's unclear to me whether you mean for our research goals, or because we can add noise later. And "for now" -- is that Model 1/Minimum Viable Model?

Reply:
GQ type should be in address already I think? I agree on adding tracked guardians!

Reply:
@SylLutze I don't think GQ type is included in address. Everyone with the same GQ type will have the same address, but we won't have a way to go backwards and tell which GQ type that address represents unless we have GQ type in the output.

Reply:
Actually, I think we know the GQ type [from household ID](https://github.com/ihmeuw/vivarium_census_prl_synth_pop/blob/419f5c434eb9558389c1420c3f135bfe0ffd5a5c/src/vivarium_census_prl_synth_pop/constants/data_values.py#L10-L23), if that is included in the output. (Right now the docs say a "random identifier" representing household which doesn't guarantee us this correspondence.)

Reply:
Okay! I will add that in more explicitly then, thanks!